SYCL/ESIMD/api/functional/common.hpp

```diff
@@ -46,6 +46,9 @@ template <typename T> bool are_bitwise_equal(T lhs, T rhs) {
 
 } // namespace details
 
+// Dummy kernel for submitting some code into device side.
```

Comment:
```suggestion
// Kernel ID class.
```

Reply:
I've applied this comment [here](https://github.com/intel/llvm-test-suite/pull/762/commits/f6c0e9809f2bbdef3c23496b8f2dd8e8c5e07a88#diff-ef0766a5c874c909fd25e4eec71893c4614c91112485144e4efcdab4d856d0a0L49-R49)